README.md

```diff
@@ -84,18 +83,20 @@ Once that setup has been completed, you can very easily call all the necessary A
 
 ```csharp
 using MobiledgeX;
-
-MobiledgeXIntegration integration = new MobiledgeXIntegration();
-bool registered = await integration.Register(); //calls Register Client
-DistributedMatchEngine.FindCloudletReply findCloudletReply = await integration.FindCloudlet(); //calls Find Cloudlet
-
-MatchingEngine dme = integration.me;
-
-Dictionary<int, AppPort> appPortsDict = dme.GetHTTPAppPorts(findCloudletReply);
-int public_port = findCloudletReply.ports[0].public_port; // if you only have one port
-AppPort appPort = appPortsDict[public_port];
-HttpClient http = await dme.GetHTTPClient(findCloudletReply, appPort, public_port, 5000);
-HttpResponseMessage message = await http.GetAsync("/"); //makes a get request
+using DistributedMatchEngine;
+
+[RequireComponent(typeof(MobiledgeX.LocationService))]
+public class YourClassName : MonoBehaviour
+{ 
+    async void Start()
+    {
+        MobiledgeXIntegration integration = new MobiledgeXIntegration();
+        integration = new MobiledgeXIntegration();
+        await integration.RegisterAndFindCloudlet();
+        integration.GetAppPort(LProto.L_PROTO_HTTP); // Get the port of the desired protocol
```

Comment:
Should this line have "AppPort appPort = "?

Reply:
Unity SDK stores the latest AppPort. If the developer wants to use a specific one though, they can do that ^